src/development/tools/sdk/release-notes/supported-platforms.md

```diff
@@ -78,17 +78,15 @@ minimal.
 |Windows |Windows 7      |
 |Linux   |Debian & below |
 
-We have dropped iOS8 support. For more information,
-see [go/rfc-ios8-deprecation] for details.
-
-[go/rfc-ios8-deprecation]: {{site.url}}/go/rfc-ios8-deprecation
-
 ### Unsupported platforms
 
 |Platform|Version               |
 |--------|----------------------|
 |Android |Android SDK 18 & below|
-|iOS     |iOS 8 & below         |
```

Comment:
Consider lining up the | 's, otherwise the more OCD members of the team might jump up and down =)

Reply:
Done.